emrun.py

```diff
@@ -1332,6 +1363,23 @@ def unwrap(s):
   return s
 
 
+def list_processes_by_name(exe_full_path):
+  pids = []
+  try:
+    import psutil
+    for proc in psutil.process_iter():
+      try:
+        pinfo = proc.as_dict(attrs=['pid', 'name', 'exe'])
+        if pinfo['exe'].lower().replace('\\', '/') == exe_full_path.lower().replace('\\', '/'):
+          pids.append(pinfo)
+      except Exception:
+        pass
+  except Exception:
+    pass
```

Comment:
What kind of thing can fail here?  I worry about catch all exception like this because it means the code can be broken without noticing it.   Can you remove or scope down the use of try/catch here?

Reply:
The intent is to fail gracefully is psutil is not able to iterate a process, or is not available.